src/Parts/InviteStageInstanceObject.php

```diff
@@ -9,7 +9,7 @@
 class InviteStageInstanceObject
 {
     /**
-     * @var \Ragnarok\Fenrir\Parts\GuildMember[]
+     * @var GuildMember[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)